### DISCUSSION TITLE
I need an example for client and server implementation

### DISCUSSION THREAD

**Discussion by MrProper5050** [FreeRDP/FreeRDP#6942] [Q&A]
I collected FreeRDP for 2 days and now I have a question. How do I make server and client using this library? I want to make my application for remote access!

**Comment by akallabeth** [comment]
Already provided with source code.
1. check the `client/Sample` and `server/Sample` for (very basic) examples and check the other implementations for some more sophisticated ones.
1. there is https://github.com/ogon-project/ogon which implements a full fledged RDP server for linux
2. there is https://gitlab.gnome.org/GNOME/gnome-remote-desktop @pnowack is working on getting the RDP backend upstream 
3. check https://www.freerdp.com/ for referenced projects utilizing freerdp

**Reply by MrProper5050** [reply]
oh thanks, it'll help me
